components/model_analyzer/dcgm/dcgm_value.py

```diff
@@ -0,0 +1,151 @@
+# Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+DCGM_INT32_BLANK = 0x7ffffff0
+DCGM_INT64_BLANK = 0x7ffffffffffffff0
+
+# Base value for double blank. 2 ** 47. FP 64 has 52 bits of mantissa,
+# so 47 bits can still increment by 1 and represent each value from 0-15
+DCGM_FP64_BLANK = 140737488355328.0
+
+DCGM_STR_BLANK = "<<<NULL>>>"
+
+# Represents an error where data was not found
+DCGM_INT32_NOT_FOUND = (DCGM_INT32_BLANK + 1)
+DCGM_INT64_NOT_FOUND = (DCGM_INT64_BLANK + 1)
+DCGM_FP64_NOT_FOUND = (DCGM_FP64_BLANK + 1.0)
+DCGM_STR_NOT_FOUND = "<<<NOT_FOUND>>>"
+
+# Represents an error where fetching the value is not supported
+DCGM_INT32_NOT_SUPPORTED = (DCGM_INT32_BLANK + 2)
+DCGM_INT64_NOT_SUPPORTED = (DCGM_INT64_BLANK + 2)
+DCGM_FP64_NOT_SUPPORTED = (DCGM_FP64_BLANK + 2.0)
+DCGM_STR_NOT_SUPPORTED = "<<<NOT_SUPPORTED>>>"
+
+# Represents and error where fetching the value is not allowed with our current
+# credentials
+DCGM_INT32_NOT_PERMISSIONED = (DCGM_INT32_BLANK + 3)
+DCGM_INT64_NOT_PERMISSIONED = (DCGM_INT64_BLANK + 3)
+DCGM_FP64_NOT_PERMISSIONED = (DCGM_FP64_BLANK + 3.0)
+DCGM_STR_NOT_PERMISSIONED = "<<<NOT_PERM>>>"
+
+
+###############################################################################
+# Functions to check if a value is blank or not
+def DCGM_INT32_IS_BLANK(val):
+    if val >= DCGM_INT32_BLANK:
+        return True
+    else:
+        return False
+
+
+def DCGM_INT64_IS_BLANK(val):
+    if val >= DCGM_INT64_BLANK:
+        return True
+    else:
+        return False
+
+
+def DCGM_FP64_IS_BLANK(val):
+    if val >= DCGM_FP64_BLANK:
+        return True
+    else:
+        return False
+
+
+# Looks for <<< at first position and >>> inside string
+def DCGM_STR_IS_BLANK(val):
+    if 0 != val.find("<<<"):
```

Comment:
feels awkward, not really sure what it's doing

Reply:
please check the above comment https://github.com/pytorch/benchmark/pull/929#discussion_r881901268